pandas/tests/arrays/masked/test_arrow_compat.py

```diff
@@ -89,12 +86,89 @@ def test_arrow_sliced(data):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.fixture
+def np_dtype_to_arrays(any_real_dtype):
```

Comment:
can you type this (mainly the output for better readability) also adding a doc-string explaining the purpose

Reply:
This is a fixture in the tests, I don't think we generally ask for annotations there? (eg in conftest.py there is no typing)